src/main/java/com/cloudbees/hudson/plugins/folder/relocate/StandardHandler.java

```diff
@@ -73,7 +77,54 @@ public HandlingMode applicability(Item item) {
         return Items.move((I) item, destination);
     }
 
-    @Override public List<? extends ItemGroup<?>> validDestinations(Item item) {
+    public boolean hasValidDestination(Item item) {
```

Comment:
This method will work correctly IFF it is invoked by user with Global Access (SYSTEM & Co). As designed?